apps/mobile/mobile-app/components/ui/skeleton/index.web.tsx

```
@@ -1,103 +1,103 @@
+					ref={ref}
+					className={`animate-pulse ${startColor} ${skeletonStyle({
+						variant,
+						speed,
+						class: className,
+					})}`}
+					{...props}
+				/>
+			)
+		} else {
+			return children
+		}
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)